src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/PackageSearchResult.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
```

Comment:
How about using System.Text.Json? If it increases the scope of this PR, then I am okay with creating a follow-up issue. I assume, in general, we would like to move to STJ because it is part of the .NET SDK, but I could be wrong here :)"

Reply:
fixed